docker/README.md

```diff
@@ -1,23 +1,23 @@
-# Run GreenGuard using Docker
+# Run Draco using Docker
 
-GreenGuard is prepared to be run using [Docker](https://docker.com/).
+Draco is prepared to be run using [Docker](https://docker.com/).
 
 These are the commands needed to start a Docker container locally that runs a [Jupyter Notebook](
-https://jupyter.org/) already configured to run GreenGuard.
+https://jupyter.org/) already configured to run Draco.
 
 ```bash
-docker run -ti -p8888:8888 signalsdev/greenguard:latest
+docker run -ti -p8888:8888 signalsdev/draco:latest
 ```
 
-This will start a Jupyter Notebook instance on your computer already configured to use GreenGuard.
+This will start a Jupyter Notebook instance on your computer already configured to use Draco.
 You can access it by pointing your browser at http://127.0.0.1:8888
 
 Further details about the usage of this image can be found [here](
-https://hub.docker.com/repository/docker/signalsdev/greenguard).
+https://hub.docker.com/repository/docker/signalsdev/draco).
```

Comment:
It seems that draco is under signalsdev?

We should also check orion and other libraries as well if they use `docker`

Reply:
For Orion, you can compile a docker instance but we did not push a container to the hub like GreenGuard. We need to update this actually.